cli/cmd/logs.go

```diff
@@ -10,13 +12,31 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var logsDescription = "Check Service logs. Requires one Service <node,electrs,esplora,chopsticks>"
+
 var LogsCmd = &cobra.Command{
-	Use:     "logs",
-	Short:   "Check service logs",
+	Args: func(cmd *cobra.Command, args []string) error {
+		_, found := controller.Services[args[0]]
+		if !found {
+			argsHelpPrint()
+			return errors.New("Requires one Service <node,electrs,esplora,chopsticks>")
+		}
+		return nil
+	},
+	Use:     "logs <service>",
+	Short:   logsDescription,
+	Long:    logsDescription,
 	RunE:    logs,
 	PreRunE: logsChecks,
 }
 
+func argsHelpPrint() {
```

Comment:
this function should return a string. This string should be consumed by other variables (ie to be concatenated)

Reply:
Call the function something like serviceList()